generators/server/templates/native-image.properties.ejs

```diff
@@ -0,0 +1,15 @@
+Args = -H:Name=<%= dasherizedBaseName %> \
+        -H:Class=<%=packageName%>.<%=mainClass%> \
+        --no-fallback \
+        --report-unsupported-elements-at-runtime \
```

Comment:
Please keep in mind that the usage of `--report-unsupported-elements-at-runtime` is discouraged and it shouldn't be used where possible. If that's the only way to create the native-image, then you should report an issue on https://github.com/oracle/graal/issues to let the GraalVM team know that it's a bug.

Reply:
Compilation fails without this flag, I'm keeping this flag for now and resolving the issue here.